docs/querying/api.md

```diff
@@ -450,7 +450,10 @@ $ curl http://localhost:9090/api/v1/rules
                                     "severity": "page"
                                 },
                                 "state": "firing",
-                                "value": 1
+                                "value": [
```

Comment:
That's not right, it should be the value as a string - not a list with a timestamp.

Reply:
This is using the Scalar type as suggested which is a timestamp + value. If we are okay with modifying the API to return a different value type I think this approach is the best solution as it mimics what the query API currently returns.

Reply:
We're technically allowed to make breaking changes here, however if we wanted the timestamp I think we'd put it in a different field.

Reply:
Why not keep is the same as the query API? 

```
$ curl -s  -g 'http://$(hostname)/prometheus/api/v1/query?query=count(up)/0'|jq .
{
  "status": "success",
  "data": {
    "resultType": "vector",
    "result": [
      {
        "metric": {},
        "value": [
          1551461908.309,
          "+Inf"
        ]
      }
    ]
  }
}
```

Reply:
That'd break existing users unnecessarily.

Reply:
Okay I understand. I thought we were moving forward with breaking the API given the clause. 

Also, sorry I've been busy with other work. Hopefully I can revisit sometime next week. If someone wants to implement this before me, feel free.